tasks/libs/common/go.py

```diff
@@ -11,8 +11,10 @@
 
 def download_go_dependencies(ctx: Context, paths: list[str], verbose: bool = False, max_retry: int = 3):
     print("downloading dependencies")
-    with timed("go mod download"):
```

Comment:
Is there a difference if we do not run `go mod download` but only `go mod tidy` ?

Reply:
I'm not sure, I can try it

Reply:
It does not seem to make a difference, I'll make a note about investigating it later.